content-src/selectors/selectors.js

```diff
@@ -0,0 +1,51 @@
+const {createSelector} = require("reselect");
+const dedupe = require("lib/dedupe");
+const getBestImage = require("lib/getBestImage");
+
+const SPOTLIGHT_LENGTH = module.exports.SPOTLIGHT_LENGTH = 3;
+
+module.exports.justDispatch = (() => ({}));
+
+const selectSpotlight = module.exports.selectSpotlight = createSelector(
+  [state => state.History],
+  (History) => {
+    const rows = History.rows
+    .map(site => {
```

Comment:
nit: is the indentation right here? I would've expected it two spaces in, but I don't really know what the accepted rule is.